tardis/io/model_reader.py

```diff
@@ -98,6 +98,45 @@ def read_abundances_file(abundance_filename, abundance_filetype,
     return index, abundances
 
 
+def read_uniform_abundances(abundances_section, no_of_shells):
+    """
+    Parameters
+    ----------
+
+    abundances_section: ~config.model.abundances
+    no_of_shells: int
+
+    Returns
+    -------
+    abundance: ~pandas.DataFrame
+    isotope_abundance: ~pandas.DataFrame
+    """
+    abundance = pd.DataFrame(columns=np.arange(no_of_shells),
+                             index=pd.Index(np.arange(1, 120),
+                                            name='atomic_number'),
+                             dtype=np.float64)
+
+    isotope_index = pd.MultiIndex(
+        [[]] * 2, [[]] * 2, names=['atomic_number', 'mass_number'])
+    isotope_abundance = pd.DataFrame(columns=np.arange(no_of_shells),
+                                     index=isotope_index,
+                                     dtype=np.float64)
+
+    for element_symbol_string in abundances_section:
+        if element_symbol_string == 'type':
+            continue
+        try:
+            z = element_symbol2atomic_number(element_symbol_string)
+            abundance.ix[z] = float(
+                abundances_section[element_symbol_string])
+        except MalformedElementSymbolError:
```

Comment:
@wkerzendorf If this fails, then it will throw error raised by `pyne`. 
(Ex- `RuntimeError: Not a Nuclide! NIEFEG --> 0`)

If I use `pyne` then I can replace this by something like this:
```
try:
      find atomic_no and mass_no by pyne
      if mass_no!=0:
           Its a isotope
      else:
           Its a normal element
except RuntimeError as e:
      raise RuntimeError('Abundances are not defined properly in config', e.args)
```
`Pyne` returns `mass_no` zero whenever we pass elements in normal form(without `mass_no` attached), something like this:
```
>>> nucname.anum('O')
0
>>> nucname.anum('Ni')
0
```